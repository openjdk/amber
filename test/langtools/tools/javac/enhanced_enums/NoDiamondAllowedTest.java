/*
 * @test /nodynamiccopyright/
 * @bug 8177505
 * @summary diamond is not allowed in enhanced enums
 * @compile/fail/ref=NoDiamondAllowedTest.out -XDrawDiagnostics NoDiamondAllowedTest.java
 */

enum NoDiamondAllowedTest<X> {
    A<>("");

    NoDiamondAllowedTest(X x) {}
}
