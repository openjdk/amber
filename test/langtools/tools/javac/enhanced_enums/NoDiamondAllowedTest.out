NoDiamondAllowedTest.java:9:7: compiler.err.illegal.start.of.type
1 error
